docs/decisions/0001-purpose-of-this-repo.rst

```diff
@@ -6,52 +6,39 @@ Status
-
 Context
 *******
 
-TODO: Add context of what led to the creation of this repo.
+Axim, formerly known as tCRIL, announced a new project to integrate Open edX and WooCommerce as an e-commerce system. The intention is to connect Open edX with a third-party e-commerce platform, enabling a straightforward implementation process and making the e-commerce service accessible for various initiatives. The request for proposal can be found here: `tCRIL Funded Contribution - WooCommerce Discovery`_.
 
-.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+For this integration project, we create a `WooCommerce Discovery document`_, which proposes the creation of a WordPress plugin and an API to generate the necessary actions on the Open edX side. The following image illustrates the expected interactions.
 
-Decision
-********
+.. image:: ../_static/enrollment-diagram.png
+  :alt: Sequence diagram of purchase using Woocommerce Integration.
 
-We will create a repository...
+Sequence diagram of purchase using Woocommerce Integration.
 
-TODO: Clearly state how the context above led to the creation of this repo.
+Decision
+********
 
-.. This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+* We have created this repository, an installable Django plugin in Open edX, to provide several endpoints related to the necessary operations on the platform to offer the e-commerce service.
+* The main objective of this API is to provide endpoints for the fulfillment and refund of enrollments.
+* We want the use of the API to be wider than integration with WooCommerce. The idea is to be useful for other integrations with e-commerce services.
 
 Consequences
 ************
 
-TODO: Add what other things will change as a result of creating this repo.
-
-.. This section describes the resulting context, after applying the decision. All consequences should be listed here, not just the "positive" ones. A particular decision may have positive, negative, and neutral consequences, but all of them affect the team and project in the future.
+* You need to install this plugin to access its functionalities.
 
 Rejected Alternatives
 *********************
 
-TODO: If applicable, list viable alternatives to creating this new repo and give reasons for why they were rejected. If not applicable, remove section.
-
-.. This section lists alternate options considered, described briefly, with pros and cons.
+* Create this API directly in the core of the platform. (However, depending on its usage, this could change).
 
```

Comment:
I would also explain here that using the enrollment API was rejected because of the permissions issue we noted in the other document.